tests/test_implicit.py

```diff
@@ -361,7 +361,9 @@ def test_id_token_skip_authorization_completely_missing_nonce(self):
         response = self.client.get(reverse("oauth2_provider:authorize"), data=query_data)
         self.assertEqual(response.status_code, 302)
         self.assertIn("error=invalid_request", response["Location"])
-        self.assertIn("error_description=Request+is+missing+mandatory+nonce+paramete", response["Location"])
+        self.assertIn(
+            "error_description=Request+is+missing+mandatory+nonce+paramete", response["Location"]
+        )  # pragma: codespell-ignore
```

Comment:
or could it may be just be fixed? ie. would test pass if it was 
```suggestion
        self.assertIn("error_description=Request+is+missing+mandatory+nonce+parameter", response["Location"])
```